modules/configuring-pipelines-with-your-own-argo-workflows-instance.adoc

```diff
@@ -0,0 +1,40 @@
+:_module-type: PROCEDURE
+
+[id="configuring-pipelines-with-your-own-argo-workflows-instance_{context}"]
+= Configuring pipelines with your own Argo Workflows instance
+
+[role="_abstract"]
+You can use your own Argo Workflows instance with {productname-short} pipelines. This configuration disables the embedded Argo Workflows controllers that are managed by the {productname-short} Operator and enables your external Argo Workflows instance.
+
+You cannot enable both the embedded Argo Workflows instance and your own Argo Workflows instance on the same cluster.
```

Comment:
I wonder if this could be emphasized more as a NOTE or some type of admonition, since its just a single sentence second paragraph? wdyt?